tests/testthat/helper_testthat_correctness_transactions.R

```diff
@@ -231,3 +231,37 @@ fct.testthat.correctness.clvfittedtransactions <- function(name.model, method, d
   fct.testthat.correctness.clvfittedtransactions.staticcov.regularization.lambda.0.no.regularization(method = method, clv.apparel.staticcov = clv.apparel.staticcov,
                                                                                                      m.fitted.static = obj.fitted.static)
 }
+
+fct.testthat.correctness.clvfittedtransactions.common <- function(fct.expectation.R, fct.expectation.Cpp, params_i, obj.fitted){
+  dt.expectation.seq <- clv.time.expectation.periods(clv.time = obj.fitted@clv.data@clv.time,
+                                                     user.tp.end = 38)
+
+  result.R <- DoExpectation(dt.expectation.seq = dt.expectation.seq, params_i = params_i,
+                            fct.expectation = fct.expectation.R, clv.time = obj.fitted@clv.data@clv.time)
+
+  result.Cpp <- DoExpectation(dt.expectation.seq = dt.expectation.seq, params_i = params_i,
+                              fct.expectation = fct.expectation.Cpp, clv.time = obj.fitted@clv.data@clv.time)
+
+  expect_equal(result.R, result.Cpp)
+}
+
+fct.testthat.correctness.clvfittedtransactions.nocov <- function(method, cdnow){
+  expect_silent(clv.cdnow <- clvdata(data.transactions = cdnow,
+                                     date.format = "ymd", time.unit = "W", estimation.split = 38,
+                                     name.id = "Id", name.date = "Date", name.price = "Price"))
+
+
+  expect_silent(obj.fitted <- do.call(method, list(clv.data = clv.cdnow, verbose = FALSE)))
+
+  return(obj.fitted)
+}
+
+fct.testthat.correctness.clvfittedtransactions.staticcov <- function(method, apparelTrans, apparelStaticCov){
+  clv.apparel.static <- fct.helper.create.clvdata.apparel.staticcov(data.apparelTrans = apparelTrans,
+                                                                    data.apparelStaticCov = apparelStaticCov,
+                                                                    estimation.split = 38)
+
+  expect_silent(obj.fitted <- do.call(method, list(clv.data = clv.apparel.static, verbose = FALSE)))
+
+  return(obj.fitted)
+}
```

Comment:
dont put fitting the model on data into separate functions. would replace at dozens of places where now the method is called inplace (ie `pnbd(clv.cdnow`). Rather move the code here to the "Expectation in Rcpp matches expecttation ... " tests

Reply:
I think I don't understand what I should change here. Would you remove the clvfittedtransactions.nocov / staticcov altogether and put the whole code inside fct.testthat.correctness.clvfittedtransactions.same.expectation.in.R.and.Cpp?